Library/OcAppleEventLib/PointerHandler.c

```diff
@@ -43,8 +44,8 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 //         most machines. Poll with 10 ms, which matches the keyboard behaviour,
 //         and also is the minimum for QEMU.
 //
-#define POINTER_POLL_FREQUENCY  EFI_TIMER_PERIOD_MILLISECONDS (10)
-#define MAX_POINTER_POLL_FREQUENCY  EFI_TIMER_PERIOD_MILLISECONDS (80)
+#define MIN_POINTER_POLL_FREQUENCY  /*EFI_TIMER_PERIOD_MILLISECONDS*/ (10 * 10000)
+#define MAX_POINTER_POLL_FREQUENCY  /*EFI_TIMER_PERIOD_MILLISECONDS*/ (80 * 10000)
```

Comment:
Why?

Reply:
The original has MulDiv function, so is not a constant expression.